README.md

```diff
@@ -48,6 +49,7 @@ cardForm.getCardNumber();
 cardForm.getExpirationMonth();
 cardForm.getExpirationYear();
 cardForm.getCvv();
+cardForm.getName();
```

Comment:
These methods are no longer correct, this was changed to `cardholderNameRequired` and `getCardholderName` respectively.